src/universal/containers/App/AppContainer.js

```diff
@@ -5,21 +5,23 @@ import {localStorageVars} from '../../utils/clientOptions';
 import loginWithToken from '../../decorators/loginWithToken/loginWithToken';
 import {ensureState} from 'redux-optimistic-ui';
 
+function mapStateToProps(state) {
```

Comment:
if this is above, let's make it a function expression. I personally always favor expressions over declarations, unless readability dictates a declaration.